### PR TITLE
Adds corpus-dir option to run_fuzzer command

### PR DESCRIPTION
**Pull request by catenacyber**
Solves #2490

### CODE REVIEW

File: infra/helper.py [hunk 1]
```diff
@@ -734,10 +736,21 @@ def run_fuzzer(args):
...
       '-v', '%s:/out' % _get_output_dir(args.project_name),
+  ]
+
+  if args.corpus_dir:
+    if not os.path.exists(args.corpus_dir):
+      print('ERROR: the path provided in --corpus-dir argument does not exist',
+          file=sys.stderr)
+      return 1
+    corpus_dir = os.path.realpath(args.corpus_dir)
+    run_args.extend(['-v', '%s:/tmp' % corpus_dir])
```
Comment (inferno-chromium) [inline infra/helper.py]: I don't think this is right, why mount to /tmp ? Also, how is this corpus getting used in run_fuzzer, like how will it run fuzzer with this corpus.

Comment (catenacyber) [reply infra/helper.py]: run_fuzzer puts the corpus in a subdirectory of /tmp (at least locally for me)
Should we mount this specific subdirectory ?

The point was to get the corpus back from the Docker to the host (and not the other way around) cf https://github.com/google/oss-fuzz/issues/2490#issuecomment-539950426

Comment (inferno-chromium) [reply infra/helper.py]: @oliverchang - can you review this, i don't remember how corpus was used in /tmp here.

Comment (catenacyber) [reply infra/helper.py]: ping ?

**Comment by inferno-chromium** [comment]
Closing this PR, will find an owner to start from this one.

**Comment by catenacyber** [comment]
Is there a way I can help ?

**Comment by inferno-chromium** [comment]
> Is there a way I can help ?

For now should be ok, will let @mbarbella-chromium ask if there are questions.

**Pull request closed without merging**